pkg/testrunner/result/types.go

```diff
@@ -57,7 +60,14 @@ type Metadata struct {
 	// ComponentDescriptor describes the current component_descriptor of the direct landscape-setup components.
 	// It is formated as an array of components: { name: "my_component", version: "0.0.1" }
 	ComponentDescriptor []*componentdescriptor.Component `json:"bom"`
-	TestrunID           string                           `json:"testrun_id"`
+
+	// Name of the testrun crd object.
+	TestrunID string `json:"testrun_id"`
+
+	// Link to the workflow in the argo ui
+	// Only set if an ingress with the name "argo-ui" for the argoui is set.
+	// Argo URL is in format: https://argo-endpoint.com/workflows/namespace/wf-name
+	ArgoUIExternalURL string `json:"argo_external_url"`
```

Comment:
Again for the kibana reason: can we shorten the field name? (would then also match `cc_meta:concourse_url`)
```suggestion
	ArgoUIExternalURL string `json:"argo_url"`
```

Reply:
sure :+1:
I wasn't which url field to use as "inspiration" cause there is also this field: `cc_meta.atc-external-url`

Reply:
You are right and the justification by pointing to another existing field is actually pointless, it's all about having short but still meaningful field names :)